envbuilder.go

```diff
@@ -1382,24 +1393,32 @@ func maybeDeleteFilesystem(logger log.Func, force bool) error {
 	return util.DeleteFilesystem()
 }
 
-func copyFile(src, dst string) error {
+func copyFile(src, dst string, mode fs.FileMode) error {
 	content, err := os.ReadFile(src)
```

Comment:
In #292 I had converted these to use billy filesystem, any reason not to do this?

Reply:
We can use fs!